src/foam/u2/view/LoginView.js

```diff
@@ -306,51 +310,56 @@ foam.CLASS({
         .tag(this.data.LOGIN)
         .add(
           this.slot(function(data$showAction, data$disclaimer, appConfig) {
-            return self.E().callIf(data$showAction, function() {
-              this
-                .start()
-                  .startContext({ data: self.data })
-                  .addClass(self.myClass('center-footer'))
-                  // first footer
+            var disclaimer = self.E().callIf(data$disclaimer && appConfig, function() {
+              this.start()
+                .addClass(self.myClass('disclaimer'))
+                .add(self.DISCLAIMER_TEXT)
+                .start('a')
+                  .addClasses([self.myClass('tc-link'), 'h600'])
+                  .add(appConfig.termsAndCondLabel)
+                  .attrs({
+                    href: appConfig.termsAndCondLink,
+                    target: '_blank'
+                  })
+                .end()
+                .add(' and ')
+                .start('a')
+                  .addClasses([self.myClass('tc-link'), 'h600'])
+                  .add(appConfig.privacy)
+                  .attrs({
+                    href: appConfig.privacyUrl,
+                    target: '_blank'
+                  })
+                .end()
+              .end();
+            });
+            return self.E().callIfElse(data$showAction,
```

Comment:
can you not just keep this as callIf then add disclaimer after it, so you always get disclaimer but only get showAction if it should show?